### ISSUE TITLE
Future improvements for Pm04 - Behavioral Cloning

### ISSUE THREAD

**Issue by jmtc7** [jmtc7/self-driving-car-nanodegree#2]
A few suggestions for further improvement of the end-to-end CNN to autonomously drive a car in the Udacity Simulator:

- Use Python Generators to feed the data without having to load all at a time.
- A deeper analysis of overfitting and try additional measures.
- Additional testing. Try dropout layers to try to reduce overfitting and increase robustness.
- Add data from other circuits to the training set.
- Try to output the car's velocity (not only the steering angle).

**Comment by jmtc7** [comment]
Closed because the project has been moved into its own independent [repository](https://github.com/jmtc7/CarND-Pm04--End-to-End-Driving-CNN). This issue is now there.
